core/platform/auth/firebase_auth_services.py

```diff
@@ -136,15 +151,16 @@ def get_auth_claims_from_request(request):
     claims = _verify_id_token(request.headers.get('Authorization', ''))
     auth_id = claims.get('sub', None)
     email = claims.get('email', None)
-    role_is_super_admin = claims.get('role', None) == 'admin'
+    role_is_super_admin = (
+        claims.get('role', None) == feconf.FIREBASE_ROLE_SUPER_ADMIN)
     if auth_id:
         return auth_domain.AuthClaims(auth_id, email, role_is_super_admin)
     return None
 
 
 def mark_user_for_deletion(user_id):
     """Set the 'deleted' property of the user with given user_id to True, and
-    disables the user's Firebase account so that they cannot use it to sign-in.
+    disables the user's Firebase account so that they cannot use it to signin.
```

Comment:
Ah, to be clear, signin is still two words: "sign in".

Reply:
Done.